hacker_Rank/May5_Answer.py

```diff
@@ -0,0 +1,20 @@
+def earliest_time(a, x):
+    print(a)
+    b = []
+    for i in range(len(a)):
+        if 1 <= a[i] <= x:
+            for j in range(len(a)):
+                if a[i] == a[j] and a[i] not in b:
+                    b.append(a[i])
+                    seconds = j
+
+    print("fianl", b)
+    return "seconds " + str(seconds) + "  " + "Number of steps:" + str(b)
+
+
```

Comment:
a=[1,2,3,2,4,5,3]
x=6
earliest_time(a,x)

output :

[1, 2, 3, 2, 4, 5, 3]
fianl [1, 2, 3, 4, 5]
'seconds 5  Number of steps:[1, 2, 3, 4, 5]'

expected output : -1

Reply:
updated

Reply:
take input :
a=[2,2,3,2,4,5,3]
x=5
earliest_time(a,x)

output : 
[2, 2, 3, 2, 4, 5, 3]
'seconds 5  Number of steps:[2, 3, 4, 5]'

expected :
 -1